src/posts/sota.md

```diff
@@ -14,17 +14,16 @@ _Last update: 2024-09-16_
 - **Games**: For many games ([Chess, Go](https://en.wikipedia.org/wiki/AlphaGo_Zero), Starcraft, Dota, [Gran Turismo](https://www.technologyreview.com/2022/07/19/1056176/sonys-racing-ai-destroyed-its-human-competitors-by-being-nice-and-fast/) etc.) the best AI is better than the best human.
 - **Working memory**: An average human can remember about 7 items (such as numbers) at a time. Gemini 1.5 Pro [can read and remember 99% of 7 million words](https://blog.google/technology/ai/google-gemini-next-generation-model-february-2024/#sundar-note).
 - **Reading speed**: A model like Gemini 1.5 Pro can read an entire book in 30 seconds. It can learn an entirely new language and translate texts in half a minute.
-- **Thinking speed**: AI models can write at speeds far surpassing any human, writing entire computer programmes in seconds.
+- **Writing speed**: AI models can write at speeds far surpassing any human, writing entire computer programmes in seconds.
```

Comment:
This seems unfair / seems to undersell their advantage. Yes, humans are slow at writing because of pens and keyboards and mechanics - but that's not why these LLMs speed is impressive. It's because they can output thoughts / language / text at a far greater speed than we can